Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -38,7 +38,7 @@ public static ISet<Skimmer<AnalyzeContext>> CreateOrRetrievedCachedSkimmer(IFile
                 AnalyzeCommand.CreateSkimmersFromDefinitionsFiles(fileSystem,
                                                                   new string[] { regexDefinitionsPath },
                                                                   tool,
-                                                                  RE2Regex.Instance);
+                                                                  IronRE2Regex.Instance);
```

Comment:
>IronRE2Regex [](http://example.com/codeflow?start=66&length=12)

Just to raise confidence, we run all functional tests against IronRE2. This creates a high degree of confidence it works: all outputs of all test targets is identical using this engine.